compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -445,15 +450,18 @@ class ComposeScene internal constructor(
                 focusedOwner?.onDismissRequest?.invoke()
             } else {
                 currentOwner.processPointerInput(event)
+                mousePressOwner = currentOwner
             }
         } else {
             focusedOwner?.processPointerInput(event)
+            mousePressOwner = focusedOwner
         }
     }
 
     private fun onMouseReleased(event: PointerInputEvent) {
```

Comment:
We will stuck in `mousePressOwner`, when we minimize the window during pressing the mouse:
```
// Press the mouse
Move
Move
Press
// Press Alt-Tab - minimize the window
Exit
// Release the mouse
// Press Alt-Tab - restore the window
Enter
```

I am thinking now, how we can fix this.

Reply:
It is an issue, but I think it is not severe (and only on Windows). Will try to look at it in another fix, which I plan to do.